pkg/manifests/yaml/api/crd.yaml

```diff
@@ -1,68 +1,144 @@
----
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    "api-approved.kubernetes.io": "https://github.com/kubernetes/enhancements/pull/1870"
+    api-approved.kubernetes.io: https://github.com/kubernetes/enhancements/pull/1870
```

Comment:
so now we bump the API we ship to 0.0.10 AND we add support for NFD as updater, all in this one PR?
If so, this is too much:
- API bump + artifacts bump should be its own PR
- support for NFD as updater should be its own PR (or two, one preparation + one for the actual support)

Reply:
OK I'll start split the PRs